gapis/api/templates/api.go.tmpl

```diff
@@ -162,14 +162,18 @@ import (
 
   type {{$name}} struct {
     data *[{{$ty.Size}}]{{$el}}
+    ϟrefID ϟapi.RefID
```

Comment:
IMO this should go away once we start using arena allocated data - pointers then become stable.

Reply:
Yes, the `RefID`s are intended primarily to avoid the `DependencyGraphBuilder` holding onto references to every object created. 

@ben-clayton Does the arena get around the problem of pointer re-use that @AWoloszyn mentioned?